cordova-plugin-appcenter-analytics/plugin.xml

```diff
@@ -60,6 +60,6 @@
         <header-file src="src/ios/AppCenterAnalyticsPlugin.h" />
         <source-file src="src/ios/AppCenterAnalyticsPlugin.m" />
 
-        <framework src="AppCenter" type="podspec" spec="" />
+        <framework src="AppCenter" type="podspec" spec="~> 1.12.0" />
```

Comment:
Just curious about why the AC framework is now tight to a specific version? Also why not doing it for all the supported modules like crashes too?

Reply:
@clpolet @dhei the `setUserId` API is available in v1.12.0 in iOS SDK, so i added the explicit requirement for that. Otherwise it's possible for Cordova to resolve lower version of the framework if it's not constrained in Podfile. For Crashes it got missed, fixed that, thanks!

Reply:
Though it should be resolved to the latest version by default when omitted I believe it won't hurt to express our intention explicitly here so everyone should be aware that `1.12` is the minimum required version here.

Reply:
@nevalenny make sense, thanks for the explanation.